packages/@aws-cdk/aws-servicecatalogappregistry/test/integ.application-associator.all-stacks-association.ts

```diff
@@ -10,6 +10,7 @@ const stack = new cdk.Stack(app, 'integ-servicecatalogappregistry-application');
 new appreg.ApplicationAssociator(app, 'RegisterCdkApplication', {
   applicationName: 'AppRegistryAssociatedApplication',
   description: 'Testing AppRegistry ApplicationAssociator',
+  stackId: 'AppRegistryApplicationAssociatorStack',
```

Comment:
Instead of just adding this field here, please convert this to the new integ test type and add a second stack with the applicationStackId supplied. The new style of tests allows for multiple test cases.